### ISSUE TITLE
Permalinks in tasks view

### ISSUE THREAD

**Issue by Semir1212** [humhub/tasks#241]
Currently permalinks are only available from the stream entry, they should also be available in the task overview: 

![image](https://github.com/humhub/tasks/assets/38326726/2ac97a45-6e53-4051-801b-785a4c16418a)

**Comment by yurabakhtin** [comment]
@luke- PR https://github.com/humhub/tasks/pull/251.

**Comment by luke-** [comment]
@yurabakhtin We have integrated the Stream Context Menu in the File Module / Wiki etc., shouldn't we do the same here?

**Comment by yurabakhtin** [comment]
@luke- Yes, we can integrate it as we did for Cfiles module. Should I start this?

**Comment by luke-** [comment]
Yes please!
